pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -76,12 +77,17 @@ def __init__(self, monitor: str = 'val_loss', min_delta: float = 0.0, patience:
                 log.info(f'EarlyStopping mode set to {self.mode} for monitoring {self.monitor}.')
 
         self.min_delta *= 1 if self.monitor_op == torch.gt else -1
+        self.best_score = torch_inf if self.monitor_op == torch.lt else -torch_inf
```

Comment:
can this be pickled??

Reply:
I believe so, this test checks for that 
https://github.com/PyTorchLightning/pytorch-lightning/pull/2391/files#diff-a0606aadf6970149ee18c4edcbcfdc04R120